### PR TITLE
Converge all test related documentation to a single place

### PR DESCRIPTION
**Pull request by yurynix**
### 🔦 Summary
Improve tests related documentation, part of #705

### 🗺️ Test Plan
Read the docs

### CODE REVIEW

File: docs/guides/running-tests.md [hunk 1]
```diff
@@ -144,3 +188,45 @@ If you want to run some code before your tests you can use one of the 3 followin
 These setup files are actually [Jests's `setupTestFrameworkScriptFile`](https://jestjs.io/docs/en/configuration#setuptestframeworkscriptfile-string)
 
 > A path to a module that runs some code to configure or set up the testing framework before each test.
+
+## Enzyme
```
Comment (ranyitz) [inline docs/guides/running-tests.md]: This Enzyme guide is pretty old. I would put it under mocha setup (so dev would understand it's related) or remove it completely.

@ronami thoughts?

File: docs/guides/running-tests.md [hunk 2]
```diff
@@ -1,9 +1,53 @@
 ---
-id: jest-setup
-title: Jest Setup
-sidebar_label: Jest Setup
+id: running-tests
+title: Runing tests
```
[the remark this answers is not in the document]
Comment (Schniz) [reply docs/guides/running-tests.md]:
```suggestion
title: Running tests
```

**Comment by ronami** [comment]
@yurynix is this still relevant?

**Comment by ronami** [comment]
Closing. Please re-open if it's still relevant.

**Pull request closed without merging**